.github/workflows/_base-server-tests.yml

```diff
@@ -62,9 +62,20 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        db: ${{ fromJson(inputs.enable-sqlite && '["mariadb", "sqlite"]' || '["mariadb"]') }}
```

Comment:
both postgres and sqlite can have a flag like to check

had disabled it earlier in favour of sqlite: https://github.com/frappe/frappe/commit/2b92073aa7041d04ef49995ea202af1c20d9057e